addons/web/static/src/views/pivot/pivot_model.js

```diff
@@ -1583,6 +1583,13 @@ export class PivotModel extends Model {
         ) {
             return value === undefined ? _t("None") : value ? _t("Yes") : _t("No");
         }
+        if (
+            fieldName &&
+            metaData.fields[fieldName] &&
+            metaData.fields[fieldName].type === "integer"
+        ) {
+            return value || "0";
+        }
```

Comment:
```suggestion
        if (fieldName && metaData.fields[fieldName]) {
             if (metaData.fields[fieldName].type === "boolean") {
                 return value === undefined ? _t("None") : value ? _t("Yes") : _t("No");
             } else if (metaData.fields[fieldName].type === "integer") {
                 return value || "0";
             }
        }
```
Not sure about the undefined stuff here either, but I'm less familiar with pivot. Maybe @Polymorphe57 would know. He's off this week, so let's keep it. He'll see the notif when coming back.